galasa-eclipse-parent/dev.galasa.eclipse/src/dev/galasa/eclipse/launcher/Launcher.java

```diff
@@ -458,13 +456,13 @@ private Manifest findManifest(String project) {
 			if (actualProject.hasNature(MAVEN_NATURE)) {
 				consoleDefault.append("This is a maven project: " + project + "\n");
 				IMavenProjectFacade mavenProjectFacade = MavenPlugin.getMavenProjectRegistry().getProject(actualProject);
-				String version = mavenProjectFacade.getArtifactKey().getVersion();
+				String version = mavenProjectFacade.getArtifactKey().version();
 				
 				IPath outputPath = mavenProjectFacade.getOutputLocation().removeLastSegments(1);
 				IResource actualOutputPath = workspaceRoot.findMember(outputPath);
 				java.nio.file.Path realOutputPath = Paths.get(actualOutputPath.getRawLocationURI());
 				
-				String artifactId = mavenProjectFacade.getArtifactKey().getArtifactId();
+				String artifactId = mavenProjectFacade.getArtifactKey().artifactId();
```

Comment:
Normally you'd expect them to deprecate the calls first. I guess we may have ignored any such warnings in the past, and now we've been bitten ? Wondering what other deprecation warnings we ignore now... gulp...